lib/repl.js

```diff
@@ -409,7 +408,7 @@ function REPLServer(prompt,
 
     // Check to see if a REPL keyword was used. If it returns true,
     // display next prompt and return.
-    if (cmd && cmd.charAt(0) === '.' && isNaN(parseFloat(cmd))) {
+    if (cmd.charAt(0) === '.' && isNaN(parseFloat(cmd))) {
```

Comment:
This won't have problems if a string doesn't get passed? If so, should the `if (cmd)` below be `cmd !== ''`?